community.html

```diff
@@ -21,7 +21,7 @@ <h2>Connect with us</h2>
     <li><a class="fediverse" href="https://floss.social/@sandstorm">Fediverse</a>
     <li><a class="github" href="https://github.com/sandstorm-io/sandstorm">GitHub</a>
     <li><a class="livechat" href="https://matrix.to/#/#sandstorm:libera.chat">Matrix</a>
-    <li><a class="devgroup" href="https://groups.google.com/group/sandstorm-dev">Mailing List</a>
+    <li><a class="devgroup" href="https://groups.io/g/sandstorm-user-group">Mailing List</a>

```

Comment:
Would it be a big hassle to update the css class name?

Reply:
In the long run what do you think about highlighting both lists here? I wouldn't want the fact that there are two lists to be confusing and not obvious.

Reply:
Will try updating the class name. I thought about it, but this section is about "connecting with the community", and this style of button doesn't leave much room for multiple choices.